docs/concepts.rst

```diff
@@ -197,3 +197,96 @@ used to indicate which resources should be DELETEd upon test completion.
 
    Steps
 	:ref:`step_reference_resource_cleanup`
+
+.. _actions:
+
+*******
+Actions
+*******
+
+Brine offers the ability to define a bundle of _actions_ which can be
+later evaluated.
+
+Restricted Official Usage
+=========================
+
+This functionality could be used to support a wide rannge
+of functionality, but functionality will be added to the core library
+conservatively in response to use cases established in practice or in
+response to opened issues. Reservation to add such features is due to
+`YAGNI <http://wiki.c2.com/?YouArentGonnaNeedIt>`_ with an additional
+concern that some of that functionality could obscure the primary intent
+of this library and foster its use for situations where a another solution
+(such as a general purpose language) may be better suited.
+
+Such functionality which is not offered by the official library can
+leverage the _actions_ featue and be implemented with a fairly small
+amount of code; more information will be provided through Articles.
+
+Supported Functionality
+=======================
+
+.. _polling:
+
+Polling
+-------
+
+For any system which may perform background work or uses a model of
```

Comment:
could be much shorter, perhaps: ```Polling repeats some actions until success, or fails them after some duration expires```. I suppose it depends on intended audience, but I would expect people `polling` to be in common use.

Reply:
This explanatory style likely stems from when polling was first added (unofficially) when a fair amount of time had to be spent going back and forth with the QA engineer at the time about which cases should expect some form of polling and which shouldn't: so I think it is ultimately the audience where this could try to provide criteria for someone who may be less familiar with the internals of the system under test.